spocc_download.R

```diff
@@ -0,0 +1,54 @@
+################################################################################
+################################################################################
+##################### Downloading Occurrences via SPOCC ########################
+################################################################################
+################################################################################
+
+# libraries
+library(spocc)
+library(sf)
+library(tidyverse)
+
+
+# Load previous pre-processed data
+spp <- read.csv("data/Nic_survey_occ_processed.csv")
+
+# make it spatial
+nic <- sf::st_as_sf(spp, coords = c("longitude", "latitude"), crs = 4326)
+table(nic$species)
+
+# Fort Collins
+foco_GMA <- sf::st_read("data/gis/ft_collins_GMA_boundary.shp")
+foco_GMA <- sf::st_transform(foco, st_crs(nic))
+
+# Plot
+plot(foco$geometry, axes = T) # boundary
+plot(nic["species"], add = T, pch = 16, pal = sf.colors(length(unique(nic$species))))
+
+
+# occ() // This can take some time with multiple species
+spp.listA = c("Agelaius phoeniceus", "Tyrannus verticalis", "Sturnella neglecta",
```

Comment:
As we think about broadening beyond the Nature in the City data, is it worth thinking through make this list without manually inputting species names? For example, we could store a csv of species and index it. E.g. for birds: https://checklist.americanornithology.org/ - not as familiar with butterflies.

Might just be more flexible, not as prone to error, and easier to add a long list as we expand this out.